modules/ml/src/main/java/org/apache/ignite/ml/genetic/functions/GAGridFunction.java

```diff
@@ -35,7 +35,6 @@
  * Responsible for providing custom SQL functions to retrieve optimization results
  */
 public class GAGridFunction {
-
     /** GAConfiguration **/
     private GAConfiguration config = null;
 
```

Comment:
for lines 52 and 63: are those lines is needed?

Reply:
Yes.  These line are required.